recipe/meta.yaml

```diff
@@ -35,7 +35,10 @@ requirements:
     - gettext  # [osx]
     - glib
     - harfbuzz
-    - libjpeg-turbo  # [win]
+    - libtiff
+    - libpng
+    - libjpeg-turbo
+    - zlib
```

Comment:
Something is making windows try to link against `z.lib` in the build env.


(My suspicion is gobject-introspection, but that is only because i don't understand what that package does)

Reply:
Seems like the same issue as over at https://github.com/conda-forge/gtk3-feedstock/pull/58#issuecomment-1456804747, so probably `gdk_pixbuf`. I haven't dug further as of yet.

Reply:
Yeah, maybe related to this comment:
https://github.com/conda-forge/conda-forge.github.io/issues/1880#issuecomment-1408733984